src/Doctrine/EntityDetails.php

```diff
@@ -55,17 +56,21 @@ public function getFormFields(): array
             }
         }
 
-        foreach ($this->metadata->associationMappings as $fieldName => $relation) {
-            if (\Doctrine\ORM\Mapping\ClassMetadata::ONE_TO_MANY !== $relation['type']) {
-                $fields[] = $fieldName;
-            }
-        }
-
         $fieldsWithTypes = [];
         foreach ($fields as $field) {
             $fieldsWithTypes[$field] = null;
         }
 
+        foreach ($this->metadata->associationMappings as $fieldName => $relation) {
+            if (\Doctrine\ORM\Mapping\ClassMetadata::ONE_TO_MANY !== $relation['type']) {
```

Comment:
It's been awhile since I looked in the code :). So, does this mean that if a field is a OneToMany to another entity, it would add an `EntityType`? Like, if we're building a `CategoryForm` and it has a `products` OneToMany property?

If so, do we need `multiple => true`? And why not add the other fields as well? Or maybe I'm missing the flow - it happens frequently ;)

Reply:
Ha! Thank you for pointing out the `!` I missed on the `!==` :). So no, leave OneToMany off 👍